share/plugins/colornotes.qml

```diff
@@ -74,18 +74,18 @@ MuseScore {
                         if (fullScore)
                               cursor.rewind(0) // if no selection, beginning of score
 
-                        while (cursor.segment && (fullScore || cursor.tick < endTick)) {
-                              if (cursor.element && cursor.element.type === Element.CHORD) {
-                                    var graceChords = cursor.element.graceNotes;
+                        while (cursor.segment() && (fullScore || cursor.tick < endTick)) {
+                              if (cursor.element() && cursor.element().type == Ms.CHORD) {
+                                    var graceChords = cursor.element().graceNotes;
                                     for (var i = 0; i < graceChords.length; i++) {
                                           // iterate through all grace chords
-                                          var graceNotes = graceChords[i].notes;
-                                          for (var j = 0; j < graceNotes.length; j++)
-                                                func(graceNotes[j]);
+                                          var notes = graceChords[i].notes;
+                                          for (var j = 0; j < notes.length; j++)
+                                                func(notes[j]);
```

Comment:
this still is reverting a recent change of mine

Reply:
Yeah, that's on purpose. colornotes.qml has changed to support new syntax.

Reply:
Not here. But I see you fixed it meanwhile, thanks ;-)